app/Http/Controllers/PaymentController.php

```diff
@@ -111,9 +111,9 @@ public function storeUser(Request $request)
         } else {
             //Assuming DuesTransaction for now
 
-            //Find DuesTransactions without payment attempts
+            //Find the most recent DuesTransaction without a payment attempt
             $transactWithoutPmt = DuesTransaction::doesntHave('payment')
-                ->where('user_id', $user->id)->first();
+                ->where('user_id', $user->id)->get()->sortBy('updated_at')->last();
```

Comment:
By calling `get()` then `last()` you're first getting all transactions for the user without payment and then getting the last. I think it would be more efficient to take out the `get()` but I haven't tested that.